src/test/java/org/folio/list/service/ListServiceTest.java

```diff
@@ -173,9 +172,32 @@ void testCreateList() {
     when(entityTypeClient.getEntityType(entity.getEntityTypeId())).thenReturn(entityType);
     when(fqlService.getFql(entity.getFqlQuery())).thenReturn(new Fql(equalsCondition));
     when(userFriendlyQueryService.getUserFriendlyQuery(equalsCondition, entityType)).thenReturn(userFriendlyQuery);
-    when(listMapper.toListDTO(entity)).thenReturn(expected);
```

Comment:
This mock behavior was making this test kind of useless, since the list creation method basically returns `listMapper.toListDTO(entity)`, so asserting stuff about the return value was kind of pointless when we're providing the return value in the test